### DISCUSSION TITLE
Change default color of Latex objects

### DISCUSSION THREAD

**Discussion by kacago22** [xournalpp/xournalpp#6081] [General]
Hello! I somehow changed the default color of the Latex objects to green and can't figure out how I did it - also cannot change it back to black. Also - as soon as I try to edit a previous Latex object it immediately turns green. 
Please help! Thank you!

**Comment by MiltonBalaOfficial** [comment]
change the color of text tool

**Comment by kacago22** [comment]
Thank you - wow it wasn't that complicated after all :)
